pages/osx/tart.md

```diff
@@ -0,0 +1,36 @@
+# tart
+
+> Build, run and manage macOS and Linux virtual machines (VMs) on Apple Silicon.
+> More information: <https://github.com/cirruslabs/tart>.
+
+- Pull a remote VM image:
+
+`tart pull {{acme.io/org/name:tag}}`
```

Comment:
```suggestion
- Pull a specific remote VM image:

`tart pull {{acme.io/org/name:tag}}`
```

Optional fix: `specific` may sound better.

Reply:
My pleasure! thanks for reviewing.

I think I'd prefer not to make the optional "specific" changes. I've deliberated on it for a while and, on reflection, I think it reads better as it is.

Reply:
That's why they are optional - you can do with them what you want. 😄